### PR TITLE
feat: first draft proposal implementation for handling invalid lanelets

### PR DESCRIPTION
**Pull request by ahmeddesokyebrahim**
## Description

This PR is one of group of PRs for providing a solution for the idea discussed [here](https://github.com/orgs/autowarefoundation/discussions/3275).

### Discussion
- [Discussion Link](https://github.com/orgs/autowarefoundation/discussions/3275)
### Issue 
- [Issue Link](https://github.com/autowarefoundation/autoware/issues/3308)
### Relevant PRs
  - [autoware_documentation](https://github.com/autowarefoundation/autoware-documentation/pull/337)
  - [autoware_common](https://github.com/autowarefoundation/autoware_common/pull/175)
  - [tier4/tier4_autoware_msgs - part I](https://github.com/tier4/tier4_autoware_msgs/pull/76) :heavy_check_mark: 
  - [tier4/tier4_autoware_msgs - part II](https://github.com/tier4/tier4_autoware_msgs/pull/87) :heavy_check_mark: 
  - [autoware_adapi_msgs](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/27) :heavy_check_mark: 
  - [autoware_universe](https://github.com/autowarefoundation/autoware.universe/pull/3000)
  - ~[tier4/tier4_ad_api_adaptor](https://github.com/tier4/tier4_ad_api_adaptor/pull/95)~

## Notes for reviewers
For compiling and testing this PR, all PRs in related links section need to be taken into consideration.
 :exclamation: **Merge sequence can be found [here](https://github.com/orgs/autowarefoundation/discussions/3275#discussioncomment-5379038)** :exclamation: 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/behavior_velocity_planner.param.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@
     launch_no_stopping_area: true
     launch_run_out: false
     launch_speed_bump: false
+    launch_invalid_lanelet: true
```
Comment (mehmetdogru) [inline autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/behavior_velocity_planner.param.yaml:14]: lets keep it false by default

Comment (ahmeddesokyebrahim) [reply autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/behavior_velocity_planner.param.yaml]: Fixed

**Review by yukkysaito — APPROVED** [review]
LGTM

**Review by mehmetdogru — APPROVED** [review]
LGTM

**Pull request merged**